src/utils/types.ts

```diff
@@ -24,6 +24,8 @@ import {
     VSC,
 } from './equipment-types';
 
+export type UrlString = `${string}://${string}` | `/${string}` | `./${string}`;
```

Comment:
I think this is not useful, it is not runtime validation and we don't use these defintion properties in the code (in parsing for example), so it is an overprecision that does not bring value.

Reply:
It does have an effect with tests because they were failing after the first version, but yes it's limited in usage as IdP settings are runtime data.

But isn't better as documentation and validation, as it's extends by [`Url` type](https://github.com/gridsuite/commons-ui/blob/auth/remove_implicit_code/src/services/apps-metadata.ts#L11) after?

Reply:
How could it make tests fail ? Which extension are you talking about ? I don't really see any benefit